lib/web_ui/lib/src/engine/navigation/url_strategy.dart

```diff
@@ -5,60 +5,12 @@
 import 'dart:async';
 
 import 'package:ui/ui.dart' as ui;
+import 'package:ui/ui_web/src/ui_web.dart' as ui_web;
 
 import '../dom.dart';
 import '../safe_browser_api.dart';
 import 'js_url_strategy.dart';
```

Comment:
My understanding is this will go away after we migrate `flutter_web_plugins` to use the Dart API directly, correct?

Reply:
Yep!